tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+env_list =
+    check
+    lint
+    py{38,39,310,311}-django3.2-wagtail4.1
```

Comment:
Going to get the next release out with 4.1 support, and then drop it shortly after.